### PR TITLE
[zh-cn] improve the translation

### PR DESCRIPTION
**Pull request by qlijin**
Small fix about 2 files:
content/zh-cn/docs/tutorials/services/source-ip.md
content/zh-cn/docs/tutorials/stateful-application/cassandra.md

### CODE REVIEW

File: content/zh-cn/docs/tutorials/stateful-application/cassandra.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ Create the Cassandra StatefulSet from the `cassandra-statefulset.yaml` file:
 使用 `cassandra-statefulset.yaml` 文件创建 Cassandra StatefulSet：
 
 ```shell
-# 如果你能未经修改地 apply cassandra-statefulset.yaml，请使用此命令
+# 如果你能未经修改地应用 cassandra-statefulset.yaml，请使用此命令
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/stateful-application/cassandra.md:191]: The original is better to talk about the code below

File: content/zh-cn/docs/tutorials/stateful-application/cassandra.md [hunk 2]
```diff
@@ -199,7 +199,7 @@ that manifest, from the folder you saved the modified version into:
 -->
 如果你为了适合你的集群需要修改 `cassandra-statefulset.yaml`，
 下载 https://k8s.io/examples/application/cassandra/cassandra-statefulset.yaml，
-然后 apply 修改后的清单。
+然后应用修改后的清单。
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/stateful-application/cassandra.md:202]: The original is better to talk about the code below.

File: content/zh-cn/docs/tutorials/stateful-application/cassandra.md (hunk 1 above)
Comment (qlijin) [reply content/zh-cn/docs/tutorials/stateful-application/cassandra.md:191]: 感觉这个地方把一个普通动词进行中英文混用不合适。在别的文档里apply也是直接翻译成'应用'。

Comment (tengqm) [reply content/zh-cn/docs/tutorials/stateful-application/cassandra.md:191]: 同意，这里翻译成“应用”是可接受的。

File: content/zh-cn/docs/tutorials/stateful-application/cassandra.md (hunk 2 above)
Comment (tengqm) [reply content/zh-cn/docs/tutorials/stateful-application/cassandra.md:202]: 同上

**Review by windsonsea — APPROVED** [review]

**Review by Sea-n — COMMENTED** [review]
/approve

**Pull request merged**